policies/schema.yml

```diff
@@ -2,163 +2,177 @@ type: map
   publisher:
     type: str
-    desc: publisher name from SHERAP/RoMEO. Do not edit
+    desc: "publisher: Publisher name from SHERAP/RoMEO. Do not edit"
   policy-heading:
     type: str
-    desc: policy heading from SHERAP/RoMEO. Do not edit
+    desc: "policy-heading: policy heading from SHERAP/RoMEO. Do not edit"
   journals:
     type: seq
-    desc: A list of the journals from SHERPA/RoMEO associated with this romeo policy. Do not edit
+    desc: "journals: A list of the journals from SHERPA/RoMEO associated with this romeo policy. Do not edit"
+    sequence:
+      - type: str
+  journals-child-policies:
+    type: seq
+    desc:  "journals-child-policies: A list of the journals from SHERPA/RoMEO associated with this policy's child policies. Do not edit"
     sequence:
       - type: str
   parent-policies:
     type: seq
-    desc: A list of the Policy IDs from SHERPA/RoMEO that are the parent of this policy. Do not edit
+    desc: "parent-policies: A list of the Policy IDs from SHERPA/RoMEO that are the parent of this policy. Do not edit"
     sequence:
       - type: str
   child-policies:
     type: seq
-    desc: A list of the Policy IDs from SHERPA/RoMEO that list this Policy ID as its parent. Do not edit
+    desc: "child-policies: A list of the Policy IDs from SHERPA/RoMEO that list this Policy ID as its parent. Do not edit"
     sequence:
       - type: str
   flag-romeo:
-    desc: Leave any notes about upstream RoMEO issues here. For example, if this policy does not apply to the journals listed above, leave a note here.
+    desc: "flag-romeo: Leave any notes about upstream RoMEO issues here. For example, if this policy does not apply to the journals listed above, leave a note here."
     type: str
   
   # OPEN PEER REVIEW
   peer-review-url:
     type: str
     pattern: "^(ht|f)tps?://\\S+$"
-    desc: Peer review policy url (valid url)
+    desc: "peer-review-url: Open peer review policy url (valid url)"
   open-reports:
     type: str
     enum: [mandatory, optional, no]
-    desc: Does the journal publish the content of peer reviews? (mandatory/optional/no)
+    desc: "open-reports: Does the journal publish the content of peer reviews? (mandatory/optional/no/not specified)"
   identities-revealed:
     type: str
     enum: [mandatory, optional, no]
-    desc: Are reviewer identities revealed to the author? (mandatory/optional/no)
+    desc: "identities-revealed: Are reviewer identities revealed to the author? (mandatory/optional/no/not specified)"
   identities-published:
     type: str
     enum: [mandatory, optional, no]
-    desc: Are reviewer identities published? (mandatory/optional/no)
+    desc: "identities-published: Are reviewer identities published? (mandatory/optional/no/not specified)"
   
   # CO-REVIEWERS
   co-review-url:
     type: str
     pattern: "^(ht|f)tps?://\\S+$"
-    desc: Co-reviewer policy url (valid url)
+    desc: "co-review-url: Co-reviewer policy url (valid url)"
```

Comment:
It looks like you've repeated the field name as the first part of the description which I don't think is actually that helpful, since the YAML files will then look like:

```yml
# co-review-url: Co-reviewer policy url (valid url)
co-review-url:
```

I thought the idea was to lead off with the section name, so the YAML files would look like:

```yml
# CO-REVIEWERS: Co-reviewer policy url (valid url)
co-review-url:
```

Thus the section a field belonged to would be obvious from it's comment. @jpolka2 what do you think?